packages/loader/container-loader/src/loader.ts

```diff
@@ -79,19 +76,10 @@ export class RelativeLoader extends EventEmitter implements ILoader {
     }
 
     public async request(request: IRequest): Promise<IResponse> {
-        const containerUrl = this.containerUrl();
         if (request.url.startsWith("/")) {
-            let container: IContainer;
-            if (canUseCache(request)) {
-                container = await this.containerDeferred.promise;
-            } else if (containerUrl === undefined) {
-                throw new Error("Container url is not provided");
-            } else {
-                container = await this.loader.resolve({ url: containerUrl, headers: request.headers });
-            }
+            const container = await this.containerDeferred.promise;
```

Comment:
we need to check `canUseCache`, and if false (like for the summarizer) we need to do Container.load and create a new container

Reply:
Why is that necessary? Shouldn't the requests need to be routed to original container anyway?

Reply:
In the resolve part also, we just return the original container for a relative request.

Reply:
no. the summarizer cannot reuse the container, as that container can have user edits

Reply:
Got it. I think in that case, I will close this PR,